systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -92,7 +91,6 @@
     @Test
     @Tag(REGRESSION)
     @OpenShiftOnly
-    @Resources(value = "../examples/templates/cluster-operator", asAdmin = true)
```

Comment:
I guess we need templates to execute `oc.newApp("strimzi-ephemeral", map("CLUSTER_NAME", clusterName));`